test/firewall_stage1_proposal_test.rb

```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env rspec
-
-require_relative "test_helper"
-
-require "yast"
-
-module Yast
-  extend Yast::I18n
-
-  import "SuSEFirewall4Network"
-  import "SuSEFirewallProposal"
-  import "WFM"
-
-  describe "FirewallStage1ProposalClient" do
```

Comment:
Deleting tests is not so nice :(

Reply:
yep, not so nice, but for pure clients it is hard to mock them and I need to mock that bootloader loading part as bootloader is not build requirement.